pkgs/by-name/se/seq-cli/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildDotnetModule,
+  fetchFromGitHub,
+  dotnetCorePackages,
+  testers,
+  seq-cli,
```

Comment:
Instead of adding the package to its own deps, use the function format of `buildDotnetModule`.

Reply:
Thanks for the input! I've applied them now as suggested